packages/govern-subgraph/src/OptimisticQueue.ts

```diff
@@ -11,309 +11,320 @@ import {
   Vetoed as VetoedEvent,
   Ruled as RuledEvent,
   EvidenceSubmitted as EvidenceSubmittedEvent,
-  OptimisticQueue as OptimisticQueueContract,
-} from "../generated/templates/OptimisticQueue/OptimisticQueue";
+  OptimisticQueue as OptimisticQueueContract
+} from '../generated/templates/OptimisticQueue/OptimisticQueue'
 import {
   Config as ConfigEntity,
   Challenge as ChallengeEntity,
   Collateral as CollateralEntity,
   Evidence as EvidenceEntity,
   Item as ItemEntity,
   OptimisticQueue as OptimisticQueueEntity,
-  Veto as VetoEntity,
-} from "../generated/schema";
-import { frozenRoles, roleGranted, roleRevoked } from "./lib/MiniACL";
-
-const NONE_STATUS = "None";
-const APPROVED_STATUS = "Approved";
-const CANCELLED_STATUS = "Cancelled";
-const CHALLENGED_STATUS = "Challenged";
-const EXECUTED_STATUS = "Executed";
-const REJECTED_STATUS = "Rejected";
-const SCHEDULED_STATUS = "Scheduled";
-const VETOED_STATUS = "Vetoed";
-
-const ALLOW_RULING = BigInt.fromI32(4);
+  ERC20 as ERC20Entity,
+  Veto as VetoEntity
+} from '../generated/schema'
+import { frozenRoles, roleGranted, roleRevoked } from './lib/MiniACL'
+
+const NONE_STATUS = 'None'
+const APPROVED_STATUS = 'Approved'
+const CANCELLED_STATUS = 'Cancelled'
+const CHALLENGED_STATUS = 'Challenged'
+const EXECUTED_STATUS = 'Executed'
+const REJECTED_STATUS = 'Rejected'
+const SCHEDULED_STATUS = 'Scheduled'
+const VETOED_STATUS = 'Vetoed'
+
+const ALLOW_RULING = BigInt.fromI32(4)
 
 export function handleScheduled(event: ScheduledEvent): void {
-  const queue = loadOrCreateQueue(event.address);
+  const queue = loadOrCreateQueue(event.address)
 
-  const item = loadOrCreateItem(event.params.containerHash, event);
+  const item = loadOrCreateItem(event.params.containerHash, event)
```

Comment:
I'm under the slight impression that is isn't working properly. AFAIK the latest subgraph version running is from this PR, and I've sent a few actions to a test DAO (`bitconnectr`) and it isn't picking up the events.

Reply:
I'll go ahead and merge this changes and then review the particular issue consider the new changes introduced on the contracts by Jorge.